src/components/worker/worker.py

```diff
@@ -517,6 +517,11 @@ async def get_pcd(self) -> CapturedData:
 
         pc_obj = msg["pcl"]
         points = pc_obj.getPoints().astype(np.float64)
+
+        # Convert to right-handed system by negating z if configured
+        if isinstance(self.cfg, OakDConfig) and self.cfg.right_handed_system:
+            points[:, 2] = -points[:, 2]
```

Comment:
Makes sense with simple flip here. Has the motion team just been doing this on the other side the API? Or is that harder to do once packed in PCD?

Reply:
that's a good point. how did we get to this point again? @nfranczak

Reply:
if the camera's coordinate system is left handed you can't add it to the framesystem

Reply:
> Has the motion team just been doing this on the other side the API?

motion team has not used this camera since it can't be meaningfully added to the framesystem.

-----

sure, we could specify some random orientation vector and then iterate through each point of a pointcloud within a client script, but that is really un-ideal.

Reply:
I did the transform with miko a long time ago - I can show you what the frame transform looks like with a frame system using your left and right hands - it actually might be flipping x for the oak-d

Reply:
On that note - this pr requires images to show what the point cloud looks like with this attribute change and without.

Reply:
Talked to Miko. We should be flipping y